.github/workflows/generate-reports.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will generate surefire reports and API documentation with Maven and publish
+# them on the 'gh-pages' branch of the target Github repository.
+
+name: Generate reports and API documentation
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - development
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install git
+        run: sudo apt-get install git
+
+      - name: Set up JDK 1.11
```

Comment:
should be java 8 everywhere? How can I introduce java 8 with qube?

Reply:
I think it should be for now. You can just change 1.11 to 1.8

Reply:
Perfect, thanks!